CSS/Homepage.css

```diff
@@ -1,12 +1,7 @@
-.header {
-    display: flex;
-    flex-direction: column;
-}
-
 .header__title {
     display: flex;
     justify-content: center;
-    margin: 50px;
+    margin: 80px 0px 40px 0px;
```

Comment:
Please Avoid margin top

Reply:
unless is the page/main container margins

Reply:
fix the margins